utils_cv/detection/plot.py

```diff
@@ -171,6 +171,16 @@ def plot_keypoints(
                 width=plot_settings.keypoint_th,
             )
 
+        # draw keypoints
+        visible_point_xys = keypoints[keypoints[..., 2] != 0][..., :2]
+        offset = 2 * plot_settings.keypoint_th
+        point_bboxes = np.hstack([
```

Comment:
"bbox" seems like the wrong variable name here, since we use it to refer to an instance of the BoundingBox class. Maybe "rect" instead for rectangle?

Reply:
Renamed